Minor formatting fixes, one code fix

bradumbaugh opened the pull request:
- Removed unnecessary indenting in the code
- Removed unnecessary styling in the checklists (the check icons aren't right)
- Fixed a code error

azurecla commented:
Hi __@msolhab__, I'm your friendly neighborhood Azure Pull Request Bot (You can call me AZPRBOT). Thanks for your contribution!
    <p>
        It looks like you're working at Microsoft (mohabib). If you're full-time, we DON'T require a contribution license agreement.
    </p>
    <p>
        If you are a vendor, or work for Microsoft Open Technologies, DO please sign the electronic contribution license agreement. It will take 2 minutes and there's no faxing! https://cla.azure.com.
    </p>

TTYL, AZPRBOT;

Commit:
Minor formatting fixes, one code fix

- Removed unnecessary indenting in the code
- Removed unnecessary styling in the checklists (the check icons aren't right)
- Fixed a code error
